### PR TITLE
[zh-cn] sync sync change-runtime-containerd configure-aggregation-layer setup-ha-etcd-with-kubeadm ingress-controllers user-namespaces

### PR DESCRIPTION
**Pull request by asa3311**
change-runtime-containerd.md
configure-aggregation-layer.md
setup-ha-etcd-with-kubeadm.md
ingress-controllers.md
user-namespaces.md

### CODE REVIEW

**Comment by howieyuen** [comment]
I'm quite curious about how do you find all these tiny changes,  run `./scripts/lsync.sh` with the root dir of `zh-cn` doc?

**Comment by asa3311** [comment]
> I'm quite curious about how do you find all these tiny changes, run `./scripts/lsync.sh` with the root dir of `zh-cn` doc?

yes，./scripts/lsync.sh content/zh-cn

**Comment by howieyuen** [comment]
OMG
/lgtm

File: content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ then run the following commands:
 <!--
 ## Configure the kubelet to use containerd as its container runtime
 
-Edit the file `/var/lib/kubelet/kubeadm-flags.env` and add the containerd runtime to the flags.
+Edit the file `/var/lib/kubelet/kubeadm-flags.env` and add the containerd runtime to the flags;
 `--container-runtime=remote` and
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md:161]: This line is removed.

**Comment by asa3311** [comment]
conflic

**Pull request closed without merging**